### ISSUE TITLE
some inconsistency in summarizing regional and county conditions

### ISSUE THREAD

**Issue by ipetrovits** [amaximus/met_alerts_hu#9]
Hi, great work there, really love this alerting integration!!

However I think i found some edge-case, my config is:

```
- platform: met_alerts_hu
  name: "met_alert"
  region_id: 1305
  county_id: 13
```

Conditions are:
![image](https://user-images.githubusercontent.com/50260134/125062200-6d713880-e0ae-11eb-961b-a0f2202401bf.png)
![image](https://user-images.githubusercontent.com/50260134/125062211-6fd39280-e0ae-11eb-952e-331e9a8a6d66.png)

And the result is:
```
alerts:
  - level: '1'
    type: Zivatar
    icon: mdi:weather-lightning-rainy
  - level: '1'
    type: Felhőszakadás
    icon: mdi:weather-pouring
  - level: '1'
    type: Hőség
    icon: mdi:weather-sunny-alert
dominant_met_alert_value: 1
dominant_met_alert: Zivatar
updated: '2021-07-09 10:00 '
nr_of_alerts: '4'
provider: Data provided by met.hu
friendly_name: met_alert
icon: mdi:weather-lightning-rainy
```

Severities could change in the last hours, but in total there are still 4 alerts (1 regional and 3 county), that is total 4 but it seems because the same type, it overwrites the level - which is more valid in the regional level I think - but does not revert the valid total number.

**Comment by amaximus** [comment]
This has been fixed this morning in version 2.2.1 already available in HACS.

Please update the card, restart HASS and get back to me with results.

**Comment by ipetrovits** [comment]
Thanks, it is resolved.
